### DISCUSSION TITLE
Help helping out testing Astroid

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#910] [Q&A] [answered]
Hello Astroid Community,

What is the best way for me to test Astroid? Maybe point me to a good tutorial for said purpose?

To date I have figure out how (in MAMP on Mac) to:

- Create clean installs of Joomla and then install pretty much anything locally in install
- Install and get the Quick Start Astroid Boiler Plate Template working
- Download my live site and work with it locally

But I just do not know where to start when eg:

- a new release is coming out and I am asked to test before final release!
(am I supposed to download a test package, some where?)

- or I am asked to test some thing specifically like eg Facebook article thumbnail functionality like here:
https://github.com/templaza/astroid-framework/commit/1ddf31174b73bf35f082791a3b2b8050a805ce01
(am I supposed to download eg the latest released offical package and replace the specific file eg script.php
 with the new more up to date version?)

Hope this makes sense and thanks for any tips :-)

**Comment by sonvnn** [comment, marked as answer]
Hi Paul 

Thanks for taking the time to test and document Astroid

Basically the latest we're working on is in v3-branch. You can download v3-branch [here](https://github.com/templaza/astroid-framework/archive/refs/heads/v3.zip). This link is always the same, never changes. So you can rest assured to use the latest version here without having to search elsewhere.

Please feel free to contact me if you need further assistance. It may take me a few days to respond as I am very busy. But I will definitely respond as soon as possible.

Best regards,
Sonny

**Comment by paurray** [comment]
hi @sonvnn
thanks for all your support and amazing work.  I will return to this early next year.
all the best
Paul
